lib/ed25519.js

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {
+  sign,
+  verify,
+  createPrivateKey,
+  createPublicKey,
+  randomBytes
```

Comment:
We don't need to expose `randomBytes`. That's only being done because stablelib needs it to be passed into its `generateKeyPair` function. But that's an implementation detail of the browser version that has no business influencing this version -- this is breaking abstraction. What we need to do is expose functions in our common `ed25519` API that do what we need (and only the common behavior that doesn't let people pass in PRNGs) ... and hide the other details behind the scenes.

Reply:
I believe this address this issue: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/11d6eb9cf0ce47777e6fe4ec4e6386a922b94ca5